openai_vector/challenges/run-benchmark.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "run-benchmark",
+    "default": true,
+    "description": "Challenge to close, restore, and benchmark index with specified operations.",
+    "schedule": [
+      {
+        "name": "close-benchmark-index",
+        "operation": {
+          "operation-type": "raw-request",
+          "path": "/vdb910-openai/_close",
+          "method": "POST"
+        }
+      },
+      {
+        "name": "restore-snapshot",
+        "operation": {
+          "operation-type": "raw-request",
+          "path": "/_snapshot/vdb910-openai/vdb910-openai/_restore",
+          "method": "POST",
+          "body": {
+            "indices": "vdb910-openai"
+          }
+        }
+      },
+      {
+        "name": "check-cluster-health",
+        "operation": "check-cluster-health"
+      },
+      {
+        "name": "wait-until-merges-finish-after-index",
+        "operation": {
+          "operation-type": "index-stats",
+          "index": "_all",
+          "condition": {
+            "path": "_all.total.merges.current",
+            "expected-value": 0
+          },
+          "retry-until-success": true,
+          "include-in-reporting": false
+        }
+      },
+      {
+        "name": "index-benchmark",
```

Comment:
since the snapshot already has the initial - index data , do we need to perform this again ?